pkg/asset/manifests/clusterapi/cluster.go

```diff
@@ -42,7 +43,8 @@ var _ asset.WritableRuntimeAsset = (*Cluster)(nil)
 // Cluster generates manifests for target cluster
 // creation using CAPI.
 type Cluster struct {
-	FileList []*asset.RuntimeFile
+	FileList   []*asset.RuntimeFile
+	ClusterIDs []string
```

Comment:
I don't think we should add this `ClusterIDs` field. An asset like this represents a file on disk, but this field will not be deserialized if we are loading the asset. It will only be populated when generating. 

It looks like you're working around this by adding a check in the Provision function to see if the field is empty, and then repopulating from the manifests. I think we should just always do that: in the Provision function gather teh cluster ID from the manifests. Can we do that or is there another reason we need the ClusterIDs field on the asset?

Reply:
Yep.  I can remove this field.  I only found I missed the "load from disk" portion of `create manifests` vs `create cluster`.  I can always just calculate this during testenv setup.

Reply:
@patrickdillon / @r4f4 , I removed the field.  Now I just pull out the cluster names when applying manifests.